github/github_test.go

```diff
@@ -1054,3 +1054,12 @@ func TestFormatRateReset(t *testing.T) {
 		t.Errorf("Format is wrong. got: %v, want: %v", got, want)
 	}
 }
+
+func TestNestedStructAccessorNoPanic(t *testing.T) {
+	issue := &Issue{User: nil}
```

Comment:
@shurcooL - what do you think about replacing this line with:

```go
var issue *Issue = nil
```

?  This is what I was envisioning in order to test the full pipeline of accessors.

Reply:
Doh.  That will crash.  Never mind.  😄

Reply:
@gmlewis I see that you had commented the following but I don't see that comment now -
> Doh. That will crash. Never mind.

Is that still valid? The only benefit I see by doing it like this is that this way, we will be testing the `nil` case even for `GetUser()` and not just for `GetPlan()`.

Reply:
> what do you think about replacing this line with `var issue *Issue = nil`

@gmlewis Both ways are equivalent. Either one will each give us full coverage.

Reply:
Yeah, I'm fine either way, actually.  I think that "Doh. That will crash. Never mind." was incorrect... so I deleted the comment... I actually think my first comment is correct, in that it tests one more nil, but I'm fine keeping it as is.  Thanks, and sorry for the noise.  :smile:

Reply:
So, if we are good with this here, should I go ahead and resolve the merge conflicts in `github-accessors.go` so that we are good to merge?

Reply:
SGTM.

Reply:
Cool! Resolved the conflicts. 👍